src/control/server/ctl_storage_rpc.go

```diff
@@ -45,7 +45,7 @@ func (c *StorageControlService) doNvmePrepare(req *ctlpb.PrepareNvmeReq) *ctlpb.
 	_, err := c.NvmePrepare(bdev.PrepareRequest{
 		HugePageCount: int(req.GetNrhugepages()),
 		TargetUser:    req.GetTargetuser(),
-		PCIWhitelist:  req.GetPciwhitelist(),
+		PCIAllowList:  req.GetPciwhitelist(),
```

Comment:
Should we bite the bullet and add support for BlockList as well? I think @shimizukko was working on that a while ago, maybe time to revive that effort?